doc-content/enterprise-only/installation/patches-applying-proc.adoc

```diff
@@ -6,8 +6,13 @@ Automated update tools are often provided with both patch updates and new minor
 
 You can use the same automated update tool to apply both patch updates and minor release upgrades to {PRODUCT} {PRODUCT_VERSION}. Patch updates of {PRODUCT}, such as an update from version {PRODUCT_VERSION} to {PRODUCT_VERSION}.1, include the latest security updates and bug fixes. Minor release upgrades of {PRODUCT}, such as an upgrade from version {PRODUCT_VERSION}.x to 7.4, include enhancements, security updates, and bug fixes.
 
+You can update or upgrade {PRODUCT} versions 7.2.x through 7.3.1 directly. For example, you can use the 7.3.0 upgrade tool to upgrade {PRODUCT} 7.2.x to version 7.3.0. Likewise, you can directly upgrade {PRODUCT} 7.2.1 to version 7.3.1. However, you must have {PRODUCT} 7.3.1 to upgrade to version 7.4.0.
```

Comment:
Likewise, you can directly upgrade {PRODUCT} 7.2.x to version 7.3.1. However, you must have {PRODUCT} 7.3.1 to upgrade to version 7.4.0.